### PR TITLE
[DC-2112] Verify row suppression of identifiable COPE survey questions

### PR DESCRIPTION
**Pull request by nishanthpp93**
- [DC-2112] Add rc/ct concepts to list
- [DC-2112] Remove unused variable
- [DC-2112] Add concepts to CT check

[DC-2112]: https://precisionmedicineinitiative.atlassian.net/browse/DC-2112?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DC-2112]: https://precisionmedicineinitiative.atlassian.net/browse/DC-2112?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DC-2112]: https://precisionmedicineinitiative.atlassian.net/browse/DC-2112?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: data_steward/analytics/rt_cdr_qc/cdr_deid_qa_report7_cope_survey.py [hunk 1]
```diff
@@ -55,26 +63,58 @@
...
 # 596884, 596885, 596886, 596887, 596888, 596889, 1310137,1333016,1310148,1310145,1310144
+
 # From DC-2109
 # 765938,765939, 765940, 765941, 765942, 765943, 765944, 765945, 765946, 765947, 765948, 765949, 765950, 765951, 765952
 
+# DC-2113
+# RT suppression
 # +
+
+rt_concepts_dict = {
```
Comment (hiro-mishima) [inline data_steward/analytics/rt_cdr_qc/cdr_deid_qa_report7_cope_survey.py:74]: This dictionary is a great idea! It looks cleaner and more intuitive.

**Review by hiro-mishima — COMMENTED** [review]
Thanks for working on this, @nishanthpp93 !
I have a question, a comment, and another comment that could be labeled as a change request.
Could you please have a look?
Thanks :)

File: data_steward/analytics/cdr_ops/controlled_tier_qc/csv/Controlled_Tier_Concept_Level.csv [hunk 2]
```diff
@@ -297,3 +297,107 @@ observation,value_as_concept_id,1585807,,YES,DC-1527
 observation,value_as_concept_id,1585808,,YES,DC-1527
 observation,value_source_value,,OrganTransplantDescription_OtherOrgan,YES,DC-1527
 observation,value_source_value,,OrganTransplantDescription_OtherTissue,YES,DC-1527
+observation,observation_source_concept_id,766006,,YES,DC-2112
```
[the remark this answers is not in the document]
Comment (nishanthpp93) [reply data_steward/analytics/cdr_ops/controlled_tier_qc/csv/Controlled_Tier_Concept_Level.csv:300]: Nice catch! I'll update this.

File: data_steward/analytics/cdr_ops/controlled_tier_qc/csv/Controlled_Tier_Concept_Level.csv [hunk 3]
```diff
@@ -297,3 +297,107 @@ observation,value_as_concept_id,1585807,,YES,DC-1527
 observation,value_as_concept_id,1585808,,YES,DC-1527
 observation,value_source_value,,OrganTransplantDescription_OtherOrgan,YES,DC-1527
 observation,value_source_value,,OrganTransplantDescription_OtherTissue,YES,DC-1527
```
[the remark this answers is not in the document]
Comment (nishanthpp93) [reply data_steward/analytics/cdr_ops/controlled_tier_qc/csv/Controlled_Tier_Concept_Level.csv:299]: That is a good point. These were built by the analytics team, so there are different structures for notebooks. There's no testing framework, but I guess any current dataset can work to see the checks are running as expected.

File: data_steward/analytics/rt_cdr_qc/cdr_deid_qa_report7_cope_survey.py (hunk 1 above)
Comment (nishanthpp93) [reply data_steward/analytics/rt_cdr_qc/cdr_deid_qa_report7_cope_survey.py:74]: Thanks, this was a mess to figure out which concept came from which ticket.

**Review by hiro-mishima — APPROVED** [review]
Thanks for updating the script, @nishanthpp93 ! I think this is good to go 🚀

**Pull request merged**